src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -706,10 +703,10 @@ Json::Value checkFee (
 
     if (fee > limit && fee != loadFee &&
         request.isMember("x-queue-okay") &&
-            request["x-queue-okay"].isBool() &&
+            request["x-queue-okay"].isConvertibleTo(Json::booleanValue) &&
```

Comment:
That's fair.  However, once permissive stuff gets into the wild it can be hard to reclaim the high road.  My weak preference is that if a field expects a `bool` it should require a `bool`.  But I'm open to the preferences of others.